requirements.txt

```diff
@@ -1,6 +1,6 @@
 distlib==0.3.1
```

Comment:
Delete this file, along with Pipfile and requirements-frozen.txt. You should just be using poetry and not pip.